src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -4733,6 +4753,188 @@ public override (int e, int f) M((int g, int h) y)
         }
 
         [Fact]
+        public void OverrideTupleMethodWithDifferentNamesInParameterType()
```

Comment:
We have a bunch of tests that verify that `ValueTuple<int, int>` and `(int, int)` are the same thing, so I didn't add explicit test.